job-tracker/src/components/TextInput/index.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+require("./style.scss");
+
+const TextInput = (props) => {
+  const { className, label, id, name, onChange, value, required } = props;
+
+  return (
+    <div className="text-input">
+      <label className="text-input-label" htmlFor={id}>
+        {label}
+      </label>
+      <input
+        className={`text-input-input ${className ? className : ""}`}
+        id={id}
+        name={name}
+        onChange={(e) => onChange(e)}
+        type="text"
+        value={value}
+        required={required}
+      />
+    </div>
+  );
+};
+
+export default TextInput;
```

Comment:
This is the nicest react component that I have seen in a long time <3

Reply:
Thank you. I forgot to add a placeholder prop, however!